synapse/handlers/account_validity.py

```diff
@@ -218,6 +329,11 @@ async def renew_account(self, renewal_token: str) -> Tuple[bool, bool, int]:
               * An int representing the user's expiry timestamp as milliseconds since the
                 epoch, or 0 if the token was invalid.
         """
+        # If a module supports triggering a renew from here, do that, otherwise do the
```

Comment:
it'd be nice to update the docstring on this method to give some allusion to the fact this is to support the legacy renewal endpoint.